charts/tvheadend/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+appVersion: release-4.2
```

Comment:
This likely needs to be 4.2

Reply:
Ah ok, I thought it should match the image tag

Reply:
The image version tag is indeed "release-4.2". Happy to change though, please confirm :)

Reply:
I think it should be the app version, not the image version. LSIO tagging is dumb

Reply:
Ahha! Thanks - learn something new every day